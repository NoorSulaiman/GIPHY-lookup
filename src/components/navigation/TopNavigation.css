#topNav{
    background-color: #683caf !important;
    margin-top: 1%;
}
