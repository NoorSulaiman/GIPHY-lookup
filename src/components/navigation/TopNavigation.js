import React from 'react';
import { Menu } from 'semantic-ui-react';
import { Link } from 'react-router-dom';
import './TopNavigation.css'

const items = [
    { key: 'home', id: 'homeLink', name: 'Home', as: Link, to: "/" },
    { key: 'favorite', id: 'favoriteLink', name: 'favorite', as: Link, to: "/favorite" },
]

const TopNavigation = () => <Menu size="big" inverted id="topNav" items={items} />


export default TopNavigation;