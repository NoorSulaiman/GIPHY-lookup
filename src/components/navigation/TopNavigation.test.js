import React from 'react';
import { shallow } from 'enzyme';
import TopNavigation from './TopNavigation';

it('should render TopNavigation correctly with given strings', () => {
    const component = shallow(<TopNavigation />);
    expect(component).toMatchSnapshot();
});