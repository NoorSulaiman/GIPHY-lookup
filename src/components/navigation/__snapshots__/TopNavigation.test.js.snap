// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TopNavigation correctly with given strings 1`] = `
<Menu
  id="topNav"
  inverted={true}
  items={
    Array [
      Object {
        "as": [Function],
        "id": "homeLink",
        "key": "home",
        "name": "Home",
        "to": "/",
      },
      Object {
        "as": [Function],
        "id": "favoriteLink",
        "key": "favorites",
        "name": "favorites",
        "to": "/favorite",
      },
    ]
  }
  size="large"
/>
`;
